src/api/helpers/decrypt/generate-buffer/index.test.ts

```diff
@@ -0,0 +1,15 @@
+import generateBufferInStringFormat from '.';
+import { FINANCIAL_DETAILS } from '../../../constants';
+
+const { STRING_ENCODING, OUTPUT_ENCODING } = FINANCIAL_DETAILS.ENCRYPTION.CIPHER;
+
+describe('api/helpers/decrypt/generate-buffer', () => {
+  const value = 'testValue';
+  const buffer = Buffer.from(value, STRING_ENCODING).toString(OUTPUT_ENCODING);
+
+  it('should return a valid buffer', () => {
```

Comment:
Assertions with following please:
* Empty string
* Not a string

Reply:
Have added test for error (empty string)

Reply:
How is the new test for a non-string value working without an eslint error? The test passes in a number to a function with the required string param - `value: string`

Concerning because this should be erroring.

Reply:
Sorry guys, these tests for testing a function when a string is not passed, when the param is a string type, do not have any value. It's like doing this:

```ts
const myFunction = (str: string) => {
  if (typeof str !== 'string) {
    return null;
  }  
};
```

Otherwise, we may as well convert this file into the only JS file (🤮) and/or test all other possible types e.g array, object, null, undefined, function etc.

Reply:
I want to test the runtime behaviour of the function in case the calling function passes `null` or `undefined` due to an unknown error will the exception be thrown or not.

I don't think we need to test all the possible permutations as you have rightly mentioned regarding type checking.

Reply:
Will look at a better way for these tests in a later PR